sale_stock_last_date/models/sale.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2021 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    last_delivery_date = fields.Datetime(
+        string="Last delivery date",
+        compute="_compute_last_delivery_date",
+        store=True
```

Comment:
In a large DB, this is going to have a hard time computing on the init? Maybe we could add a init hook to ease it?

Reply:
Oh, this one is still here... You should pre-create the column and leave it empty for all ones IMO.

Reply:
Or do an equivalent SQL for doing all at once.